src/api/user/dto/user.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsNotEmpty, IsString, MaxLength } from 'class-validator';
+
+export class UserDto {
+  @IsString()
+  @IsNotEmpty()
+  @MaxLength(120)
+  public username: string;
+
+  @IsString()
+  @IsNotEmpty()
+  @MaxLength(60)
```

Comment:
Beware magic numbers 👻 ! Why is a max length of 60 important? Passwords seem important are there other validations that we might want here?